README.md

```diff
@@ -1,14 +1,80 @@
 [![Build Status](https://dev.azure.com/mseng/VSIoT/_apis/build/status/Azure%20Digital%20Twins/vscode-azure-digital-twins.nightly?branchName=develop)](https://dev.azure.com/mseng/VSIoT/_build/latest?definitionId=10125&branchName=develop)
 
-# DTDL editor for Visual Studio Code
+# DTDL Editor for Visual Studio Code
 
-This extension adds support to the [Digital Twins Definition Language](https://aka.ms/DTDL), with the next features:
+## Overview
+The [Digital Twin Definition Language](https://aka.ms/DTDL) (DTDL) is a language for describing models for Plug and Play devices, device digital twins, and logical digital twins. Broadly, modeling enables IoT solutions to provision, use, and configure digital twins of all kinds from multiple sources in a single solution. Using DTDL to describe any digital twin’s abilities enables the IoT platform and IoT solutions to leverage the semantics of each digital twin.
 
-- Syntax validation.
+With the [DTDL extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=vsciot-vscode.vscode-dtdl) , you can read and write documents using DTDL more efficiently taking full advantage of the following key features:
+
+- Create interfaces from the command palette with predefined or customized templates. 
 - Intellisense to help you with the language syntax (including auto-completion).
-- Create interfaces from the command palette.
+- Use predefined code snippets to develop DTDL efficiently. 
+- Syntax validation.
+
+## Get Started
+
+### Create Interface
+
+You could use the command palette to create interface from predefined or customized templates. 
+
+- In Visual Studio Code, select **View > Command Palette** to open the VS Code command palette.
+- In the command palette, enter and run the command **DTDL: Create Interface**. 
+- Follow the instruction to assign the interface name. 
+- If there are multiple templates existing, choose a template you need. Otherwise, the extension will the basic template as default.
+- A JSON file will be created in the current folder. The file name is based on the interface name you assigned. 
+- You should replace the **{company}** field in @id with your own company’s name. Note that @id is the path component of the [Digital Twin Model Identifier](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md#digital-twin-model-identifier-dtmi) (DTMI) and should follow the DTMI rule to uniquely identify the device model.
```

Comment:
type, AFAIK this release create the ID without the `{` in the DTMI

Reply:
update the wording as "... with your own company’s name without braces ..."